### PR TITLE
Adding our login and support for JWT

### PR DESCRIPTION
**Pull request by Bojo992**
Created our login and added support for JWT.
Login:
image

Registr:
image

Refresh JWT:
image

Delete User:
image

Test:
image

Data base schema:
image

### CODE REVIEW

**Comment by Bojo992** [comment]
[Setup with JWT](https://trello.com/c/1k29mwgL/24-setup-with-jwt)

**Review by Blake1412 — DISMISSED** [review]

File: security/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - "3306:3306"
+  java-app:
+    image: security
+    container_name: app
+    depends_on:
+      - mysql
+    networks:
+      - net
+    ports:
+      - "8080:8080"
```
Comment (oakwoodwolf) [inline security/docker-compose.yml]: Services should have different ports from each other.
This port would clash with the chat service port as both are 8080

**Review by plutowootoe — DISMISSED** [review]

File: security/docker-compose.yml (hunk 1 above)
Comment (Bojo992) [reply security/docker-compose.yml]: sad-cat

**Review by oakwoodwolf — APPROVED** [review]
All works now. Tests are sufficient

**Review by Blake1412 — APPROVED** [review]

**Pull request merged**